Source/CheckoutAPIClient.swift

```diff
@@ -90,7 +90,9 @@ public class CheckoutAPIClient {
         
         let dateProvider = DateProvider()
         let framesEventLogger = FramesEventLogger(checkoutEventLogger: checkoutEventLogger, dateProvider: dateProvider)
-        let networkFlowLoggerFactory = NetworkFlowLoggerFactory(framesEventLogger: framesEventLogger)
+        let networkFlowLoggerFactory = NetworkFlowLoggerFactory(
```

Comment:
Is this right formatting ?
```
let networkFlowLoggerFactory = NetworkFlowLoggerFactory(framesEventLogger: framesEventLogger,
                                                                                                        publicKey: publicKey)
```

Reply:
This is different from what we do on 3DS, but I would prefer this style anyway. Can we adopt something similar going forwards?